feature/posts/src/test/kotlin/com/alxnophis/jetpack/posts/ui/composable/PostDetailScreenshotTest.kt

```diff
@@ -0,0 +1,85 @@
+package com.alxnophis.jetpack.posts.ui.composable
+
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onRoot
+import com.alxnophis.jetpack.posts.ui.composable.provider.PostDetailPreviewProvider
+import com.github.takahirom.roborazzi.captureRoboImage
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.GraphicsMode
+
+@GraphicsMode(GraphicsMode.Mode.NATIVE)
+@RunWith(RobolectricTestRunner::class)
+internal class PostDetailScreenshotTest {
+    @get:Rule
+    val composeRule = createComposeRule()
+    val previewProvider = PostDetailPreviewProvider()
+
+    @Test
+    fun postDetailScreenState0() {
```

Comment:
Ideas to improve test naming if worth it.

Reply:
Is possible iterate or do something to create the screenshot test using the preview provider in place to create a method for each preview provider index?